### ISSUE TITLE
Error when using ./bin/place-order

### ISSUE THREAD

**Issue by grossman8** [blockworks-foundation/mango-explorer#36]
```
~/mango/mango-explorer/bin# ./place-order --market SOL/USDC --quantity 0.01 --price 90.00 --side BUY --order-type LIMIT --account-address 2WkWHY1u7LyXuYZSjA2SEqNiZDyhr8VMrNQ8rfVtBxfk
2022-01-29 03:31:43 ⓘ SpotMarketOp Placing SOL/USDC order « Order BUY for 0.01000000 at 90.00000000 [ID: 0 / 0] LIMIT».
Traceback (most recent call last):
  File "/root/mango/mango-explorer/bin/./place-order", line 41, in <module>
    placed = market_operations.place_order(order)
  File "/root/mango/mango-explorer/mango/spotmarketoperations.py", line 177, in place_order
    place: CombinableInstructions = self.market_instruction_builder.build_place_order_instructions(
  File "/root/mango/mango-explorer/mango/spotmarketoperations.py", line 89, in build_place_order_instructions
    return build_spot_place_order_instructions(self.context, self.wallet, self.group, self.account,
  File "/root/mango/mango-explorer/mango/instructions.py", line 664, in build_spot_place_order_instructions
    open_orders_address = create_open_orders.signers[0].public_key
IndexError: list index out of range
```

It appears that the array of signers is empty.

**Comment by OpinionatedGeek** [comment]
Apologies. I've fixed this in commit 5a36608b3de5a34e72223d0b61dbb936d380a743 and it'll be packaged in the next release.

The cause of this was a move to using PDAs for spot openorders - I updated the code everywhere but I missed this one spot. Sorry.

**Comment by grossman8** [comment]
Thanks, looks like it's fixed in 3.3.3
